src/jquery.remodal.js

```diff
@@ -199,14 +201,14 @@
         remodal.td = tdBg > remodal.td ? tdBg : remodal.td;
 
         // Add close button event listener
-        remodal.$closeButton.bind("click." + pluginName, function(e) {
+        remodal.$closeButton.bind("click." + remodal.settings.baseClass, function(e) {
```

Comment:
You changed the namespace of the events too. So i have an idea, let's rename `baseClass` to `namespace`.